context/app/routes_main.py

```diff
@@ -141,3 +141,16 @@ def showcase_view(name):
         title='Showcase',
         flask_data=core_props
     )
+
+@blueprint.route('/collections')
+def collections():
+    if 'nexus_token' not in session:
+        abort(403)
```

Comment:
```suggestion
```
Login shouldn't be necessary; but if it doesn't work right now without a token, let's get in touch with Bill and hear what the plan is.